flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/DataStream.scala

```diff
@@ -144,6 +145,43 @@ class DataStream[T](stream: JavaStream[T]) {
     this
   }
 
+
+  /**
+   * Returns the minimum resource of this operation.
+   */
+  def getMinResource: ResourceSpec = stream.getMinResource()
```

Comment:
To make getters feel more like proper Scala, I would write this as
```scala
def minResource: ResourceSpec = stream.getMinResource()
```

The "get" keyword is something that Java encourages, but Scala discourages.